.env.default

```diff
@@ -1,8 +1,9 @@
 SERVER_HOSTNAME="localhost"
-SERVER_PORT="3000"
+SERVER_PORT="3001"
 LOG_LEVEL="debug"
 APPLICATION_NAME="io-openid-provider"
 REDIS_URL="redis://0.0.0.0:6379"
 REDIS_KEY_PREFIX="oidc:"
 TEST_CLIENT_ID="example-client"
-TEST_CLIENT_REDIRECT_URI="https://relying-party.io/callback"
+TEST_CLIENT_REDIRECT_URI="https://a604-93-43-201-145.ngrok.io/callback"
```

Comment:
Is this change required due to the demo?

Reply:
It's an error. I'm going to put back the original value.